app.js

```diff
@@ -31,191 +35,235 @@ app.use(express.json({ verify: VerifyDiscordRequest(process.env.PUBLIC_KEY) }));
 // Store for in-progress games. In production, you'd want to use a DB
 const activeGames = {};
 
-/**
- * Client commands that run recurrently (Crons)
- */
-initializeDiscordClient();
-const daily = new CronJob(
-    "30 0 0 * * *",
-    postDailyMessages,
-    null, // onComplete
-    true, // autostart
-    null, // timeZone
-    null, // context
-    null, // runOnInit
-    0, // utcOffset
-);
-console.log("daily cron has started and will next run at: ", daily.nextDates(1)[0].toUTC(0).toISO());
-
-const weekly = new CronJob(
-    "0 0 20 * * 0",
-    postWeeklyMessages,
-    null, // onComplete
-    true, // autostart
-    null, // timeZone
-    null, // context
-    null, // runOnInit
-    0, // utcOffset
-);
-console.log("weekly has started and will next run at: ", weekly.nextDates(1)[0].toUTC(0).toISO());
+const discordClient = await initializeDiscordClient();
 
 /**
- * Interactions endpoint URL where Discord will send HTTP requests
+ * Client commands that run recurrently (Crons) and blocking
  */
-app.post("/interactions", async (req, res) => {
-    // Interaction type and data
-    const { type, id, data } = req.body;
+const runDaemon = async () => {
```

Comment:
can we have a way to also kill these crons?

I think sometimes we might want to restart or hard stop something that it's running recurrently, maybe to triage an issue, I think something like killDaemon would be helpful.

Reply:
I think so, but that should be done in a separate issue in my opinion. This was just to create the interface. We also need to find a way to kill the agent, because no matter what it's leaving the app in blocking mode!

Reply:
Agree. I reported issue #59 to remember this.